.zenodo.json

```
@@ -0,0 +1,34 @@
+{
+  "creators": [
+    {
+      "affiliation": "810 Labs",
+      "name": "Arsenovic, Alex",
+      "orcid": "0000-0002-8599-5873"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Hadfield, Hugo",
+      "orcid": "0000-0003-4318-050X"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Wieser, Eric",
+      "orcid": "0000-0003-0412-4978"
+    },
+    {
+      "affiliation": "Enthough",
+      "name": "Kern, Robert"
+    },
+    {
+      "name": "The Pygae Team, "
```

```suggestion
      "name": "The Pygae Team"
```
Sure the comma is a good idea?

---

This was predicting bibtex problems with spaces in words... attempted to set it all as a second name, although I might be wrong and it might just work!

---

```suggestion
      "name": "{The Pygae Team}"
```
might work for zenodo, if it doesn't attempt to escape it.